invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/modal.overrides

```diff
@@ -18,8 +18,36 @@
 
 #community-selection-modal {
 
+  .ui.segment.community-list-container {
+    padding: 0;
+  }
+
   .community-list-results.content {
     height: calc(55vh - 10em);
+
+    .item {
+      padding: 2rem !important; //overriding semantic ui's !important
```

Comment:
```suggestion
      padding: @defaultPadding !important; //overriding semantic ui's !important
```
can the usage of `!important` be avoided in this case?

Reply:
SUI is using !important on `.item:first-child`, so `!important` is needed, but I changed it to be applied only to the `:first-child` element.